### PR TITLE
feat(service-version): adjust docs [tdx-2281]

### PR DESCRIPTION
**Pull request by tomaszwylezek**
Signed-off-by: Tomasz Wylężek <tomwylezek@gmail.com>

### Summary

Adjust content related to service version lifecycle.
For now the work is behind FF and when the feature is GA then we will merge that docs adjustment

### Reason
TDX-2281

### Testing

Definitely want layout / readability / if proper place opinions about what I wrote.
https://deploy-preview-4334--kongdocs.netlify.app/konnect/servicehub/service-versions/

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@tomaszwylezek Is this content ready for a review? If everything is finalized, could you change it from the draft state to ready for review? After we review it, we won't merge it until you give us the OK. Thanks!

**Comment by acgoldsmith** [comment]
update from discussion w/ Sunny: this is expected to GA before Summit.

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:75]:
```suggestion
1. Click  **Service version actions** > **Edit version status**.
```

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:77]:
```suggestion
1. Select a service version status.
```

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:79]: @tomaszwylezek What other steps would there be here?

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:81]:
```suggestion
1. Click **Save**.
```

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:87]:
```suggestion
1. From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select the service version you want to delete.
```

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:89]:
```suggestion
1. Click **Service version actions** > **Delete service version**.
```

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:90]:
```suggestion
1. Confirm the deletion by typing the name of the service version and clicking **Yes, delete**.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
Suggested a few small changes.

File: app/konnect/servicehub/service-versions.md
Comment (rosskukulinski) [reply app/konnect/servicehub/service-versions.md:30]: I think we should also indicate that Dev Portal users with applications registered to this service version will receive an email notification that this version has been deprecated.

**Comment by cloudjumpercat** [comment]
@tomaszwylezek Could you review my suggestions when you have the time? Thanks!

File: app/konnect/servicehub/service-versions.md
Comment (tomaszwylezek) [reply app/konnect/servicehub/service-versions.md:79]: In the case of changing the version status into "deprecated" - we have one more step in which the user is displayed information about that there will be - check video here - https://github.com/Kong/khcp-ui/pull/2642

**Comment by tomaszwylezek** [comment]
@cloudjumpercat hey, 
First -  I want to apologize to you that I did it just now - I must have missed that notification somehow. 😿 
Second - Introduced all of yours suggestions and answered questions

File: app/konnect/servicehub/service-versions.md
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:81]:
```suggestion
    1. Deprecation only: Click **Save** on the dialog to deprecate the service version and send consumers an email notification.
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Comment by cloudjumpercat** [comment]
@tomaszwylezek No worries! Do you know when this will go GA? Just wondering when I should publish this. I know they said around Summit time, but I didn't know which day that would be exactly. Thanks!

**Comment by tomaszwylezek** [comment]
@cloudjumpercat 
I just talked with @nateslo about it. He will check and confirm when it should be published

**Comment by cloudjumpercat** [comment]
@tomaszwylezek @nateslo Hi! Just checking in to see if these docs can be published now that Summit is over. Thanks!

**Comment by nateslo** [comment]
> @tomaszwylezek @nateslo Hi! Just checking in to see if these docs can be published now that Summit is over. Thanks!

Hi @cloudjumpercat , we are planning on enabling the feature by the end of the week. We are doing a little more testing in the next day or so.

**Pull request merged**
